clients/drcachesim/simulator/cache_simulator.h

```diff
@@ -42,8 +42,20 @@
 #include "cache_stats.h"
 #include "cache.h"
 #include "snoop_filter.h"
+#include <limits.h>
 
-enum cache_split_t { DATA, INSTRUCTION };
+enum class cache_split_t { DATA, INSTRUCTION };
+
+// Error codes returned when passing wrong parameters to the function for
+// accessing cache statistics.
```

Comment:
Use the name `get_cache_metric` rather than a description so a reader knows precisely which one